controller/ingress_entry.go

```diff
@@ -28,8 +28,8 @@ type LoadBalancerEntry struct {
 }
 
 // FilterInvalidEntries returns a slice of all the valid LoadBalancer entries
```

Comment:
Ingresses rather than LoadBalancer. 

Do we want each Updater to validate rather than the controller or do you think this is applicable for all?